README.md

```diff
@@ -64,7 +64,9 @@ Enable it for a node with editable text like this:
             inline:
               editorOptions:
                 hyphens: true
-                
+
+This will add a new button to insert a soft hyphen. As an alternative, you can use the shortcut `Control + Shift + -` (Mac) / `Strg + Shift + -` (Windows/Unix) to add a new soft hyphen. To change the keys of the shortcut, have a look at the [Settings.yaml](Configuration/Settings.yaml) file.
```

Comment:
Is `Control` not the same as `Strg` (Steuerung)?
So it's the same on both systems right?

Reply:
You are right. But most shortcuts, that say `Ctrl` mean `CMD` on a Mac. But i can update the Readme if you want to.

Reply:
If it's command it should say CMD. And Strg should in any case be CTRL.